src/js/generate-entries.sh

```diff
@@ -42,10 +42,10 @@ cd "$root"/tools/ts-generator/types &&
 
 cd "$root"/tools/ts-generator/rpc &&
   $py rpc_gen_js.py \
-    --server-define-file "$root"/src/idl/coproc_idl.json \
+    --server-define-file "$root"/src/v/coproc/gen.json \
     --output-file "$root"/src/js/modules/rpc/serverAndClients/server.ts
 
 cd "$root"/tools/ts-generator/rpc &&
   $py rpc_gen_js.py \
-    --server-define-file "$root"/src/idl/process_batch_coproc.json \
+    --server-define-file "$root"/src/v/coproc/gen_engine.json \
```

Comment:
`v` was deprecated. This should use `redpanda`

Reply:
Same for the other path.

Reply:
`https://github.com/vectorizedio/redpanda/tree/dev/src` I'm checking `dev` branch and I don't see `redpanda` folder, where is that change?

Reply:
@0x5d its the v folder underneath source, not the root that used to be named /v , seems ok